tests/testthat/test-diagnosands.R

```diff
@@ -226,36 +223,71 @@ test_that("diagnosis, NAs if no estimand", {
   d <- declare_population(sleep) + ols
   
   sleep_ols <-
-    structure(list(design_label = structure(1L, .Label = "d", class = "factor"), 
-                   estimator_label = "estimator", term = "group2", bias = NA_real_, 
-                   `se(bias)` = NA_real_, rmse = NA_real_, `se(rmse)` = NA_real_, 
-                   power = 0, `se(power)` = 0, coverage = NA_real_, `se(coverage)` = NA_real_, 
-                   mean_estimate = 1.58, `se(mean_estimate)` = 0, sd_estimate = 0, 
-                   `se(sd_estimate)` = 0, mean_se = 0.849091017238762, `se(mean_se)` = 0, 
-                   type_s_rate = NaN, `se(type_s_rate)` = NA_real_, mean_estimand = NA_real_, 
-                   `se(mean_estimand)` = NA_real_, n_deleted = 0, `se(n_deleted)` = 0, 
-                   n_sims = 4L), class = "data.frame", row.names = c(NA, -1L
-                   ))
+    structure(
+      list(
+        design_label = structure(1L, .Label = "d", class = "factor"),
+        estimator_label = "estimator",
+        term = "group2",
+        bias = NA_real_,
+        `se(bias)` = NA_real_,
+        rmse = NA_real_,
+        `se(rmse)` = NA_real_,
+        power = 0,
+        `se(power)` = 0,
+        coverage = NA_real_,
+        `se(coverage)` = NA_real_,
+        mean_estimate = 1.58,
+        `se(mean_estimate)` = 0,
+        sd_estimate = 0,
+        `se(sd_estimate)` = 0,
+        mean_se = 0.849091017238762,
+        `se(mean_se)` = 0,
+        type_s_rate = NaN,
+        `se(type_s_rate)` = NA_real_,
+        mean_estimand = NA_real_,
+        `se(mean_estimand)` = NA_real_,
+        n_sims = 4L
+      ),
+      class = "data.frame",
+      row.names = c(NA,-1L)
+    )
```

Comment:
this chunk was generated by dput, no need to format it.

Reply:
thanks fixed